content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -196,6 +196,109 @@ add a brief description in the :guilabel:`Description` field. Click :guilabel:`F
 the :guilabel:`Rule Configuration` wizard. Now, the new rule is active and devices can connect to
 the Windows virtual :abbr:`IoT (Internet of Things)` box.
 
+Worldline exception
+~~~~~~~~~~~~~~~~~~~
+
+Worldline is a payment terminal that can be connected to Odoo's point of sale system. It allows for
+a comprehensive and fluid payment experience for customers. Worldline is available in Benelux
+(coalition of Belgium, the Netherlands, and Luxembourg).
+
+When using the Windows IoT server to connect the Worldline payment terminal, it is necessary to create
+an exception in the Windows firewall so that a connection can be made between the Odoo
+database/:abbr:`IoT (Internet of Things)` box and Worldline.
+
+.. seealso::
+   :doc:`../../../sales/point_of_sale/payment_methods/terminals/worldline`
+
+To create the exception, first, open the Windows Defender Firewall app on the Windows machine. This
+can be accomplished by going to the :guilabel:`Search bar` and typing in `windows defender`.
```

Comment:
```suggestion
To create the exception, first, open the *Windows Defender Firewall* app on the Windows machine.
This can be accomplished by typing `windows defender` in the :guilabel:`Search` bar.
```